### PR TITLE
Removes /obj/item/melee.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
As far as I can tell, this type is literally unused.

### CODE REVIEW

**Review by comma — DISMISSED** [review]

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -213,15 +213,15 @@
...
+		/obj/item/baton,
+		/obj/item/telebaton,
 		/obj/item/flame/lighter,
 		/obj/item/flashlight,
 		/obj/item/modular_computer/pda,
 		/obj/item/radio/headset,
 		/obj/item/hailer,
 		/obj/item/megaphone,
-		/obj/item/melee,
+		/obj/item,
```
Comment (Techhead0) [inline code/game/objects/items/weapons/storage/belt.dm]: This is probably a little too permissive

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -245,15 +245,15 @@
...
```
Comment (Techhead0) [inline code/game/objects/items/weapons/storage/belt.dm]: (As above)

File: code/game/objects/items/weapons/melee/energy.dm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/obj/item/melee/energy
+/obj/item/energy
```
Comment (Techhead0) [inline code/game/objects/items/weapons/melee/energy.dm]: `/obj/item/melee/energy` gives you a fair grasp of what you're talking about, `/obj/item/energy` does not

Perhaps `/obj/item/eblade` or something

Comment (MistakeNot4892) [reply code/game/objects/items/weapons/melee/energy.dm]: `/obj/item/lightsaber`

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/game/objects/items/weapons/storage/belt.dm]: Hurk I thought I got all of these. Will update.

**Review by comma — DISMISSED** [review]

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**